lib/widgets/action_sheet.dart

```diff
@@ -253,10 +253,16 @@ class BottomSheetEmptyContentPlaceholder extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final designVariables = DesignVariables.of(context);
-
+    final loc = ZulipLocalizations.of(context);
```

Comment:
```suggestion
    final zulipLocalizations = ZulipLocalizations.of(context);
```

(Like everywhere else we do this.)